RELEASING.md

```diff
@@ -0,0 +1,21 @@
+# Release process
+
+Testcontainers' release process is automated as a Travis deployment. This describes the basic steps for a project member to perform a release.
+
+## Steps
+
+1. Ensure that the master branch is building and that tests are passing.
+1. Check that the [`CHANGELOG`](CHANGELOG.md) file is up to date and includes all merged features.
```

Comment:
Since the modifications of the CHANGELOG.md files aren't automated yet, we could add that you have to update the CHANGELOG file as well (moving the points in unreleased to the new version).